pandas/api/io/__init__.py

```diff
@@ -0,0 +1,2 @@
+""" Public IO API """
+from pandas.formats.style import Styler  # noqa
```

Comment:
``pandas.io.api`` imports * to ``pandas`` namespace; so you *could* make ``Styler`` just show up there (or you can leave the ``from pandas.io.api import *``  and just ``del Styler`` (prob shorter)